server/templates/staff/course/assignment/assignment.publish.html

```diff
@@ -34,6 +34,7 @@ <h3 class="box-title">Publish Scores</h3>
 
           </div>
           <div class="box-body">
+            {{ assignment.published_scores }}
```

Comment:
We can make this slightly nicer? Like `Published scores: ', '.join(kind.title() for kin in assignment.published_scores)` or something